Automattic/wp-calypso
# Media: Remove media library selected items store and data component

#### Changes proposed in this Pull Request

* Media: media library selected items and related data component

#### Testing instructions

* Verify that the removed code isn't used anywhere.
* Verify that tests pass.
* No further testing should be necessary.

Note: #40273 precedes this PR and should be merged first. Feel free to review this one separately, though.

Note: This PR is part of an iteration to remove the media library selected items store, see #40269 for the full effort that removes it completely. Also, see #39379 for the complete media reduxification effort.
